code/__HELPERS/unsorted.dm

```diff
@@ -1579,3 +1579,11 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 	return call(source, proctype)(arglist(arguments))
 
 #define TURF_FROM_COORDS_LIST(List) (locate(List[1], List[2], List[3]))
+
+/proc/get_box(corner_x, corner_y, z, size_x, size_y)
+	. = list()
+	for	(var/x in corner_x to corner_x + size_x)
```

Comment:
just curious if i'm missing something obvious, but why not just use block()?

Reply:
... i didn't know it existed lol